lib/rules/web/admin_console/4.4/monitoring_alerts.xyaml

```diff
@@ -0,0 +1,112 @@
+click_icon_index_button:
+  element:
+    selector:
+      xpath: (//button[@aria-label='<aria_label_text>'])[<index>]
+    op: click
+set_placeholder_input_value:
+  element:
+    selector:
+      xpath: (//input[@placeholder='<placeholder_text>'])[<index>]
+    op: send_keys <input_value>
+    type: input
+clear_label_input_value:
+  elements:
+  - selector:
+      xpath: //label[text()='<label_text>']/..//input
+    op: clear
+    type: input
+set_label_input_value:
+  element:
+    selector:
+      xpath: //label[text()='<label_text>']/..//input
+    op: send_keys <input_value>
+    type: input
+check_div_text:
+  element:
+    selector:
+      xpath: //div[contains(text(),'<text>')]
+
+open_alert_detail:
+  action: click_alert_link_with_text
+silence_alert_from_action:
```

Comment:
It can be defined as 
```
silence_alert_from_action:
  params:
    item: Silence Alert
  action: click_one_dropdown_action
  action: click_create_button
```

Reply:
changed as the following:
`perform_action_item:
  params:
    button_text: <item_text>
  action: click_actions_button
  action: click_button_text
silence_alert_from_action:
  params:
    item_text: Silence Alert
  action: perform_action_item
  action: click_create_button`